app/Http/Controllers/Front/CheckoutController.php

```diff
@@ -228,9 +228,12 @@ public function postCheckout(Request $request)
                     $attributes = $this->getAttributes($content);
                 }
             } else {
+                
                 $items = new \Illuminate\Support\Collection();
                 $invoiceid = $request->input('invoice_id');
                 $invoice = $this->invoice->find($invoiceid);
+                $processingFee = $payment_method =="razorpay"? 0 : $request->input('process_fee');
+                $invoice->grand_total = intval($invoice->grand_total+($invoice->grand_total*$processingFee/100)) ;
```

Comment:
can simplify it to `intval($invoice->grand_total*(1+$processingFee/100))`

Reply:
Fixed